app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+// Entry point for the build script in your package.json
+import '@hotwired/turbo-rails';
+import './controllers';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
```

Comment:
- [ ] Good job so far 👍🏽. I think you should pay closer attention to the react-redux part of your app. You imported an `App `component here which does not exist in your app, and this `App.js `component was supposed to contain the routes of your `Greeting.js` component which is also missing. Kindly check it out as this is what is required in the project requirement 😄 .